OpenXmlPowerTools/PtOpenXmlUtil.cs

```diff
@@ -743,66 +745,38 @@ public static int CalcWidthOfRunInTwips(XElement r)
                 runText = sb.ToString();
             }
 
+            int w;
```

Comment:
The next block seems like it could be moved into the `MetricGetter.GetTextWidth` with an appropriate default return value (ie, as mentioned above, handle all possible errors there)